### ISSUE TITLE
[Question]: How do I perform animatediff tile-upscale

### ISSUE THREAD

**Issue by ShmuelRonen** [continue-revolution/sd-webui-animatediff#232]
### Expected behavior

Hello, and thank you for you work!!

I work with animatediff-cli-prompt-travel and there is the animatediff tile-upscale option.
How do I implement it here?

Example:
animatediff tile-upscale "W:\animatediff-cli-prompt-travel\output\2023-10-19T19-20-05-anime-incredibleworld_v20\00-2529226679637802235" -c config/prompts/anime.json -W768 -H1152

**Comment by continue-revolution** [comment]
Try using ControlNet tile directly with source video frame path as control images (put the path to frames into AnimateDiff video path or ControlNet batch tab), if that’s gonna work then that is tile-upscale. If that’s not gonna work, we will have to read the source code.

**Comment by ShmuelRonen** [comment]
I will check it. Is it possible to put the PNG files of each video in a separate folder and not have them all together, for example, in the outputs\txt2img-images\2023-10-19 folder?

**Comment by continue-revolution** [comment]
Go to Settings/AnimateDiff and you will know which setting you should check

**Comment by ShmuelRonen** [comment]
Thanks! now the PNG files is in a separete folders.
When I do an upscale  in animatediff-cli-prompt-travel, Besides the PNG files, it uses the creation script (json) to create the upscale with  tile-upscale.
How can I solve it in sd-webui-animatediff?

Example: 
[prompt.json](https://github.com/continue-revolution/sd-webui-animatediff/files/13046392/prompt.json)

**Comment by continue-revolution** [comment]
I don't think this config has anything else you need to config in webui. Just use ControlNet tile directly and see what's going on.

**Comment by ShmuelRonen** [comment]
OK. 
 As you wrote, I try using ControlNet tile directly with source video frame path as control images (put the path to frames into AnimateDiff video path or ControlNet batch tab), if that’s gonna work then that is tile-upscale.
The original video was 512x512 and I changed to 1024x1024. it's works for me but take a very long time (25 min for 16 frames) on my RTX 3090. 
Original:
![01235-3229817589](https://github.com/continue-revolution/sd-webui-animatediff/assets/80190186/338d0fec-8577-4b18-8881-2b02af095c92)
Tiled:
![01236-359361637](https://github.com/continue-revolution/sd-webui-animatediff/assets/80190186/7e16d4f2-1db4-4f4d-93b7-366b1adbea4c)

Now I try to do tiled-upscale on animatediff-cli-prompt-travel with the same PNG files. now it's take 2 min only for the same resoults.!!!
Tiled:
![00_5667687773227342612_Alberto-Vargas-1990s-East-LA-chola_1990s-East-LA-c](https://github.com/continue-revolution/sd-webui-animatediff/assets/80190186/4f44abee-a0fb-43f6-8508-4be77db967a6)

Can you make it better?

**Comment by continue-revolution** [comment]
Thanks for your report. I will investigate.

**Comment by ShmuelRonen** [comment]
You should use control_v11f1e_sd15_tile and not control_v11u_sd15_tile !!!

**Comment by davedaverave** [comment]
What's the difference between the first tiled and second tiled upscale video? how were able to make it render in less time??

**Comment by ShmuelRonen** [comment]
control_v11u_sd15_tile is an older version and control_v11f1e_sd15_tile is the new one.

Example control_v11f1e_sd15_tile:
![01236-359361637](https://github.com/continue-revolution/sd-webui-animatediff/assets/80190186/eb91dc10-4b79-4e7c-8cf4-b8dbbd615cd6)

Example control_v11u_sd15_tile:

![01238-4260294787](https://github.com/continue-revolution/sd-webui-animatediff/assets/80190186/a55740ba-a060-4e08-9e0e-dc595b1c06fb)


The processing time taken by both is pretty much the same.

**Comment by ShmuelRonen** [comment]
I noticed that if the tiled-upscaling process does not use the original prompt, the proposal comes out distorted.

**Comment by ShmuelRonen** [comment]
OK. here a test that I made.

This is the 512x512 product:

![01345-973020119](https://github.com/continue-revolution/sd-webui-animatediff/assets/80190186/0bfa1791-0734-41dc-9c34-4a4ad3fa13e4)


This is 1024x1024 tiled-upscale compare (with animatediff-cli-prompt travel)
You can see that the not prompted upscale is with distortions from the original.


https://github.com/continue-revolution/sd-webui-animatediff/assets/80190186/44dedd10-cf87-44a2-a2c5-34e3bbcecb07

**Comment by ShmuelRonen** [comment]
One more test that I did...
Adjusting taild-upscale to the prompt.

Test No 1, upscale with automatic 1111 controlnet tile-upscale. it's take 17 min and the prompt commands are not kept.


https://github.com/continue-revolution/sd-webui-animatediff/assets/80190186/df95eeec-587b-4459-a5b0-fd664e7bf73a


Test No 2, upscale with animatediff-cli-pt tile-upscale. it's take 3 min and the prompt commands are kept.



https://github.com/continue-revolution/sd-webui-animatediff/assets/80190186/129353e6-8e8c-401d-aa4d-b75a22378cdb

**Comment by ShmuelRonen** [comment]
Is it possible to add tiled-upscale as part of the GUI?

**Comment by continue-revolution** [comment]
V2V VRAM problem should have been resolved. Let me know if problem still exists.

**Comment by ShmuelRonen** [comment]
Hey...

I did a comprehensive experiment regarding Tile upscale in the updated version.
**prompt:** kid need seed to grow.

In text2image mode without Animatediff, the Tile upscale from 512X512 to 1024X1024 went quickly and successfully. See an example.

512x512 :
![00001-1871256357](https://github.com/continue-revolution/sd-webui-animatediff/assets/80190186/619d8147-ae92-4943-a9a6-b42f2e53dea4)

1024x1024:
![00005-1451207930](https://github.com/continue-revolution/sd-webui-animatediff/assets/80190186/40556f68-86ef-4781-ac2c-023fa343bcce)

Now I made a short gif (16 frames) in 512x512 with animatediid web-ui
![00001-4036676523](https://github.com/continue-revolution/sd-webui-animatediff/assets/80190186/822582d1-2778-4378-aa81-a9e3bfa84d01)

When I try to resample it to 1024x1024 with tile-resample it's said the time it would take was around an hour.
![1 hour](https://github.com/continue-revolution/sd-webui-animatediff/assets/80190186/c0c34914-f282-48ac-ba54-fac7840310be)

Here I gave up and did not continue.

Am I doing something wrong?
